bibigrid/core/startup.py

```diff
@@ -121,29 +122,100 @@ def run_action(args, configurations, config_path):
     return exit_state
 
 
-def main():
+# def main():
+#     """
+#     Interprets command line, sets logger, reads configuration and runs selected action. Then exits.
+#     @return:
+#     """
+#     logging.basicConfig(format=LOGGER_FORMAT)
+#     # LOG.addHandler(logging.StreamHandler())  # stdout
+#     LOG.addHandler(logging.FileHandler("bibigrid.log"))  # file
+#     args = command_line_interpreter.interpret_command_line()
+#     set_logger_verbosity(args.verbose)
+
+#     configurations = configuration_handler.read_configuration(LOG, args.config_input)
+#     if not configurations:
+#         sys.exit(1)
+#     configurations = configuration_handler.merge_configurations(
+#         user_config=configurations,
+#         default_config_path=args.default_config_input,
+#         enforced_config_path=args.enforced_config_input,
+#         log=LOG
+#     )
+#     if configurations:
+#         sys.exit(run_action(args, configurations, args.config_input))
+
+
+# if __name__ == "__main__":
+#     main()
+
+@click.command()
+@click.option('-h', '--help', is_flag=True, help='Show this message and exit.')
+@click.option('-v', '--verbose', is_flag=True, help='Enable verbose output.')
```

Comment:
argparse's count action used to control the level of verbosity should be https://click.palletsprojects.com/en/stable/options/#id11 in click